app/helpers/login.py

```diff
@@ -10,11 +10,13 @@ def login(user):
       'data': {
         'status': 'success',
         'message': 'Successfully logged in.',
-        'auth_token': auth_token
+        'auth_token': auth_token,
+        'refresh-token': refresh_token
```

Comment:
need to add this in the create_account.py file also. and same thing here with dash vs underscore